compairifuel-api/pom.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.compairifuel</groupId>
+    <artifactId>compairifuel-api</artifactId>
+    <version>1.0</version>
+    <name>compairifuel-api</name>
+    <packaging>war</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.release>17</maven.compiler.release>
+        <junit.version>5.9.2</junit.version>
+        <swagger-ui.version>5.17.14</swagger-ui.version>
```

Comment:
We're not using this property. I think we should remove it until we work on swagger again.
``` suggestion
```